### PR TITLE
Ask for name instead of username

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: src/components/RoomDialog.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ function RoomModal({
           </div>
           <div className="RoomDialog-usernameContainer">
             <label className="RoomDialog-usernameLabel" htmlFor="username">
-              Username:
+              {t("labels.name")}
```
Comment (j-f1) [inline src/components/RoomDialog.tsx:81]: Does it make sense to have an ESLint rule banning JSXText nodes so this mistake doesn’t happen again?

Comment (lipis) [reply src/components/RoomDialog.tsx:81]: Is there a rule like that?! I don't want to add plugins for it..

Comment (j-f1) [reply src/components/RoomDialog.tsx:81]: https://eslint.org/docs/rules/no-restricted-syntax

Comment (lipis) [reply src/components/RoomDialog.tsx:81]: Let's add it.

Comment (j-f1) [reply src/components/RoomDialog.tsx:81]: https://github.com/excalidraw/excalidraw/pull/1367

**Pull request merged**